cluster/ci/config/prow/config.yaml

```diff
@@ -1,5 +1,6 @@
 plank:
   allow_cancellations: true
+  job_url_prefix: 'https://openshift-gce-devel.appspot.com/build/'
   job_url_template: 'https://openshift-gce-devel.appspot.com/build/origin-ci-test/{{if eq .Spec.Type "presubmit"}}pr-logs/pull{{else if eq .Spec.Type "batch"}}pr-logs/pull{{else}}logs{{end}}{{if ne .Spec.Refs.Repo "origin"}}/{{.Spec.Refs.Org}}_{{.Spec.Refs.Repo}}{{end}}{{if eq .Spec.Type "presubmit"}}/{{with index .Spec.Refs.Pulls 0}}{{.Number}}{{end}}{{else if eq .Spec.Type "batch"}}/batch{{end}}/{{.Spec.Job}}/{{.Status.BuildID}}/'
```

Comment:
Looks like `job_url_prefix` is from kubernetes/test-infra#9211, and that it it is set we exit [here][1] before hitting the code consuming `job_url_template`.  If that's right, then I think we should have dropped this `job_url_template` line in this commit.

[1]: https://github.com/kubernetes/test-infra/pull/9211/files#diff-c406bba679631cdc24dec3be9d68eea6R560